src/sinks/file/mod.rs

```diff
@@ -74,10 +86,16 @@ impl GenerateConfig for FileSinkConfig {
     }
 }
 
-#[derive(Deserialize, Serialize, Debug, Eq, PartialEq, Clone, Copy)]
+/// Compression algorithm.
+// TODO: Why doesn't this already use `crate::sinks::util::Compression`? :thinkies:
```

Comment:
Older and never updated to use the new utils? Open a tech-debt issue?

Reply:
Fineeeeee: opened https://github.com/vectordotdev/vector/issues/13694. :D